doc/source/extended_documentation/calibration/rainforests_calibration/rainforests_calibration.rst

```diff
@@ -132,27 +97,16 @@ Model training
 
 The model training process is relatively simple and involves collating a series of
 forecast-observation pairs with the associated feature variables into a single pandas
-dataframe.
-
-Using this dataframe, a series of lightGBM binary-classification models are trained against
-the truth values for exceedances of the forecast error relative to a series of error-threshold
-values, resulting in one model per error threshold.
-
-As the calibration method works on a per realization basis, the errors used in training
-must be representative of the underlying systematic model biases for a single ensemble
-member rather than those of the ensemble as a whole. This requires the use of a single
-representative ensemble forecast member in training. It also requires the underlying
-weather type associated with the forecast and observation in the forecast-observation
-pair be as consistent as possible.
-
-For the forecast values used in training, we use the control at the shortest available
-lead-times that distinctly represent of the underlying weather types. Specifically, we
-take the earliest 24-hour period for daily accumulations, and all lead-times over the
-earliest 24-hour period for sub-daily accumulations to capture the full cycle of diurnal
-variations.
+dataframe. In general, each ensemble member yields a separate row of the dataframe (although for 
+reasons of computational efficiency it may be desirable to only use a subset of members, for example 
+by using only the control realization in each ensemble). 
```

Comment:
I thin k this is much better. I'd recommend dropping the brackets:
```suggestion
dataframe. In general, each ensemble member yields a separate row of the dataframe, although for 
reasons of computational efficiency it may be desirable to only use a subset of members, such as the
control realization only.
```

Reply:
done